tests/e2e/pkg/kubernetes.go

```
@@ -282,6 +291,21 @@ func GetNamespace(name string) (*corev1.Namespace, error) {
 
 // CreateNamespace creates a namespace
 func CreateNamespace(name string, labels map[string]string) (*corev1.Namespace, error) {
+
+	existingNamespace, err := GetNamespace(name)
+	if existingNamespace != nil && existingNamespace.Name == name {
+		return existingNamespace, nil
+	}
+
+	if errors.IsForbidden(err) && len(os.Getenv("TEST_KUBECONFIG")) > 0 {
```

This gives me concerns that we continue to execute commands but then ignore errors when running with less privileges.  I feel like this should be done differently, not entirely clear where/how these functions are called so don't have a concrete suggestion offhand.

---

fixed, basically check for `TEST_KUBECONFIG` as first thing and don't proceed with create ns/delete ns logic. For this to work, we need to give `get` permission to testrole